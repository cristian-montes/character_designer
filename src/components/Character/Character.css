.head{
    height: "150";
    width: "150";
    background-size: 'cover';
    background-repeat: 'no-repeat';
}

.middle{
    height: "150";
    width: "150";
    background-size: 'cover';
    background-repeat: 'no-repeat';
}

.botton{
    height: "150";
    width: "150";
    background-size: 'cover';
    background-repeat: 'no-repeat';
}