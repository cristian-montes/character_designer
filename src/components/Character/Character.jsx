import './Character.css';


export default function Character({head}){
    return(
        <section>
            <article className='head'
                style={{
                    backgroundImage: `url(./${head}-head.png)`,
                    height: 200,
                    width: 200,
                    backgroundSize: 'cover',
                    backgroundRepeat: 'no-repeat',
                  }}
            >
                
            </article>
        </section>
    )
}