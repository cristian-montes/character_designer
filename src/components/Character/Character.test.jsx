import {screen, render} from '@testing-library/react';
import Character from './Character';


it('Should render the Character Component', async()=>{
    const {container} = render(<Character head='alien' middle='hoodie' bottom='jeans'/>);

    const tag = await screen.findAllByRole('article');

    expect(container).toMatchSnapshot()
    expect(tag[0]).toBeInTheDocument();
});