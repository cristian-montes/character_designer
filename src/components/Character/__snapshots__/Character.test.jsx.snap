// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the Character Component 1`] = `
<div>
  <section
    style="display: flex; flex-direction: column;"
  >
    <article
      class="head"
      style="background-image: url(./alien-head.png); height: 310px; width: 310px; background-size: cover; background-repeat: no-repeat;"
    />
    <article
      class="middle"
      style="background-image: url(./hoodie-middle.png); height: 200px; width: 310px; background-size: cover; background-repeat: no-repeat;"
    />
    <article
      class="bottom"
      style="background-image: url(./jeans-bottom.png); height: 200px; width: 310px; background-size: cover; background-repeat: no-repeat;"
    />
  </section>
</div>
`;
